notebooks/Time_Series/Basic Time Series Plotting Exercises.ipynb

```diff
@@ -0,0 +1,203 @@
+{
```

Comment:
That was intentional; the goal was to give them a skeleton of code to start with. They still need to add appropriate data-reading and imports, and then they're supposed to refactor the skeleton code to something more "loopy".